case_studies/dc2_cataloging/run_great_lake/run_bliss_machine.sbatch

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
+#SBATCH --mail-user=pduan@umich.edu
+#SBATCH --mail-type=BEGIN,END,FAIL
+
+#SBATCH --account=regier0
+#SBATCH --partition=spgpu
+
+#SBATCH --nodes=1
+#SBATCH --ntasks=1
+#SBATCH --gpus-per-node=a40:1
+#SBATCH --cpus-per-gpu=16
+#SBATCH --mem-per-gpu=32GB
+
+## wall time hours:minutes:seconds
+#SBATCH --time=16:00:00
+
+###   Load software modules
+
+module load singularity/4.1.2 cuda/12.1.1 cudnn/12.1-v8.9.0
+module list
+
+####  Commands your job should run follow this line
+
+echo "running on: $SLURM_JOB_NODELIST"
+
+EXP_POSTFIX=$1
+echo "experiment postfix is: $EXP_POSTFIX"
+
+EXP_CUR_PATH=`pwd`
+echo "current directory is: ${EXP_CUR_PATH}"
+
+echo "create output folder if not exists"
+mkdir -p ./output
+echo "create overlay image folder if not exists"
+mkdir -p ./overlay_imgs
+
+echo "copy dc2local from remote server to /tmp_data/"
+mkdir -p /tmp_data/pduan/dc2local/
+rsync -av --info=progress2 \
+ -e "ssh -i ~/.ssh/jeffbox_great_lake_side_rsa" \
+ pduan@deeplearning-01.stat.lsa.umich.edu:/data/scratch/dc2local/dc2_split_results.tar \
+ /tmp_data/pduan/dc2local/
```

Comment:
Did this turn out to be faster than copying from `/scratch`? This morning I'm getting about 4 GB/minute when copying from `/scratch` and from `/nfs/turbo/lsa-regier/lsstdesc-public/dc2/`.

Reply:
It actually may be slower to copy from deeplearning-01: iirc that computer is connected by a 100 Mbit/sec ethernet cable, which can only transfer 0.75 GB/minute at best.

Reply:
when i test this command on great lake, it can achieve at most 110MB/s and download the data within about 12 min. Yesterday, i tested copying data from `/scratch`, but it seemed to be extremely slow (more than 30 min, probably due to rush hour).

Reply:
We may be limited by the ethernet speed regardless of where we fetch the data from. 1000 Mb/s ethernet cable, which I think is what deeplearning-01 uses, can transmit at most 7.5 GB/minute.

Reply:
yes, but may be this is tolerable